enterprise-suite/templates/prometheus.yaml

```diff
@@ -93,20 +111,6 @@ spec:
               scheme: HTTP
             initialDelaySeconds: 30
 
-        - name: prometheus-server-configmap-reload
-          image: jimmidyson/configmap-reload:{{ .Values.configMapReloadVersion }}
-          args:
-            - -volume-dir=/etc/config
-            - -volume-dir=/etc/rules
-            - -webhook-url=http://localhost:9090/-/reload
-          volumeMounts:
-            - name: config-volume
-              mountPath: /etc/config
-              readOnly: true
-            - name: rules-volume
-              mountPath: /etc/rules
-              readOnly: true
```

Comment:
We still need this for the prometheus template, right? Otherwise if someone does a helm update where only the template changes, they won't get the changes. Can the configmap-reload kill/restart the pod? That would be simple enough for us I think.

Reply:
This one is the prometheus reload, which is no longer needed.

There's another task for es-monitor-api to reload configs, but I think we could do that built in without needing a side car like this?  Anyway, it isn't a common path, don't think we need reload for things to work properly in the beta.